classes/user/UserSettingsDAO.inc.php

```diff
@@ -33,10 +34,11 @@ function &getSetting($userId, $name, $journalId = null) {
 	 * @param $name string
 	 * @param $value mixed
 	 * @param $type string
+     * @param $assocType bits
 	 * @param $journalId int
 	 * @return DAOResultFactory matching Users
 	 */
-	function &getUsersBySetting($name, $value, $type = null, $journalId = null) {
+	function &getUsersBySetting($name, $value, $type = null, $assocType = null, $journalId = null) {
```

Comment:
Ah, never mind my previous comments, I see you've changed this (presumably to solve a strict mode PHP warning).

It doesn't make sense to have the `assocType` column here if it's always fixed as `ASSOC_TYPE_JOURNAL`, so I'd suggest renaming the parent implementation to `_getUsersBySetting`, and leaving the parameter list here unchanged.

Reply:
(Also for similar changes, e.g. the delete function)